### DISCUSSION TITLE
"Unsubscribe from our emails" link not available on mobile

### DISCUSSION THREAD

**Discussion by benji011** [supabase/supabase#3111] [Questions] [answered]
I checked my account preferences but all it gives me is the option to change my first name, last name & theme preference (see image below)

<img src="https://user-images.githubusercontent.com/6195689/132312449-7a308607-d023-4e62-9e81-cf5fc3e36efe.png" width="600px"/>

**Comment by soedirgo** [comment, marked as answer]
You should have a link to *Unsubscribe from our emails* at the bottom of any of the emails.

**Reply by benji011** [reply]
Got it.
But, Just to add that this link is only available on Desktop but for mobile its hidden. See images below.

### Mobile
<img src="https://user-images.githubusercontent.com/6195689/132326109-4e0a3083-df7f-44a6-ae06-a887430ddf15.jpeg" width="600px"/>

### Desktop
<img src="https://user-images.githubusercontent.com/6195689/132326222-ff7197db-7ce9-4c45-93e7-3817fa261415.png" width="600px"/>

**Reply by soedirgo** [reply]
Weird - it shows up on my phone (iOS) but I guess the HTML gets borked on some displays. We'll keep this in mind.
